syncer/syncer.go

```diff
@@ -888,16 +880,15 @@ func (s *Syncer) checkWait(job *job) bool {
 	return false
 }
 
-func (s *Syncer) saveTablePoint(db, table string, location binlog.Location) {
-	ti, err := s.schemaTracker.GetTable(db, table)
-	if err != nil && table != "" {
+func (s *Syncer) saveTablePoint(table *filter.Table, location binlog.Location) {
+	ti, err := s.schemaTracker.GetTable(table)
+	if err != nil && table.Name != "" {
```

Comment:
why check `table.Name != "" ` here?

Reply:
I don't know either. Wait other reviewers to see.

Reply:
https://github.com/pingcap/dm/pull/1201 to remove a log